types/wx-js-sdk-browser/index.d.ts

```diff
@@ -0,0 +1,774 @@
+// Type definitions for non-npm package wx-js-sdk 1.6, this is the lts version for types/wx-js-sdk-dt, which is related to wx-js-sdk 1.2
+// Project: https://developers.weixin.qq.com/doc/offiaccount/OA_Web_Apps/JS-SDK.html
+// Definitions by: Xu Anhai <https://github.com/adndyxu>
+//                 Bian Zhongjie <https://github.com/agasbzj>
+//                 Ming Huang <https://github.com/forzoom>
```

Comment:
do you need these users here? I usually try to avoid situation of creating owners on DT 
without explicit ack by the listed account owner:
https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners
this is usually reserved by action of yourself (adding/removing your own handle here)

Reply:
Yep, u are right, these guys may not interested in maintaining the repo.